src/Symfony/Component/Messenger/EventListener/SendFailedMessageForRetryListener.php

```diff
@@ -70,7 +70,18 @@ public function onMessageFailed(WorkerMessageFailedEvent $event)
             $delay = $retryStrategy->getWaitingTime($envelope, $throwable);
 
             if (null !== $this->logger) {
-                $this->logger->error('Error thrown while handling message {class}. Sending for retry #{retryCount} using {delay} ms delay. Error: "{error}"', $context + ['retryCount' => $retryCount, 'delay' => $delay, 'error' => $throwable->getMessage(), 'exception' => $throwable]);
+                $logLevel = 'error';
```

Comment:
should we use the `LogLevel` constants from psr/log here ?

Reply:
Done 👍🏼